docs/templates/snakemake/snakemake.yaml

```diff
@@ -0,0 +1,13 @@
+name: snakemake
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
```

Comment:
Probably best to hardcode versions so that everyone uses the same setup

Reply:
done